app/models/verification/base.rb

```diff
@@ -70,6 +70,10 @@ def validate_and_create_audits?(params)
     true
   end
 
+  def metadata(_params)
+    nil
+  end
```

Comment:
Send nothing by default. Although there may be value in changing this to provide a list of the scanned source plates.